--- a/modules/generated/v1.30.nix
+++ b/modules/generated/v1.30.nix
@@ -6302,7 +6302,7 @@ let
         };
         "ports" = mkOption {
           description = "List of ports to expose from the container. Not specifying a port here DOES NOT prevent that port from being exposed. Any port which is listening on the default \"0.0.0.0\" address inside a container will be accessible from the network. Modifying this array with strategic merge patch may corrupt the data. For more information See https://github.com/kubernetes/kubernetes/issues/108255. Cannot be updated.";
-          type = (types.nullOr (coerceAttrsOfSubmodulesToListByKey "io.k8s.api.core.v1.ContainerPort" "name" [ "containerPort" "protocol" ]));
+          type = (types.nullOr (coerceAttrsOfSubmodulesToListByKey "io.k8s.api.core.v1.ContainerPort" "name" [ "containerPort" ]));
           apply = attrsToList;
         };
         "readinessProbe" = mkOption {
@@ -6995,7 +6995,7 @@ let
         };
         "ports" = mkOption {
           description = "Ports are not allowed for ephemeral containers.";
-          type = (types.nullOr (coerceAttrsOfSubmodulesToListByKey "io.k8s.api.core.v1.ContainerPort" "name" [ "containerPort" "protocol" ]));
+          type = (types.nullOr (coerceAttrsOfSubmodulesToListByKey "io.k8s.api.core.v1.ContainerPort" "name" [ "containerPort" ]));
           apply = attrsToList;
         };
         "readinessProbe" = mkOption {
